### PR TITLE
[diff.cpp23.library] Note new headers in C++26

### PR DESCRIPTION
**Pull request by AlisdairM**

### CODE REVIEW

File: source/compatibility.tex [hunk 1]
```diff
@@ -40,8 +40,9 @@
 New functionality.
 \effect
 The following \Cpp{} headers are new:
-\libheaderrefx{hazard_pointer}{hazard.pointer.syn}, and
-\libheaderrefx{rcu}{rcu.syn}.
+\libheaderrefx{hazard_pointer}{hazard.pointer.syn},
+\libheaderrefx{rcu}{rcu.syn}, and
```
Comment (JohelEGP) [inline source/compatibility.tex:44]:
```suggestion
\libheaderref{rcu}, and
```

File: source/compatibility.tex [hunk 2]
```diff
@@ -216,10 +217,12 @@
 \libheaderrefx{flat_map}{flat.map.syn},
 \libheaderrefx{flat_set}{flat.set.syn},
 \libheaderref{generator},
+\libheaderrefx{mdspan}{mdspan.syn},
```
Comment (JohelEGP) [inline source/compatibility.tex:220]:
```suggestion
\libheaderref{mdspan},
```

File: source/compatibility.tex [hunk 3]
```diff
@@ -216,10 +217,12 @@
...
 \libheaderref{print},
 \libheaderref{spanstream},
-\libheaderref{stacktrace}, and
-\libheaderref{stdatomic.h}.
+\libheaderref{stacktrace},
+\libheaderref{stdatomic.h}, and
+\libheaderrefx{stdfloat}{stdfloat.syn}.
```
Comment (JohelEGP) [inline source/compatibility.tex:225]:
```suggestion
\libheaderref{stdfloat}.
```

**Review by JohelEGP — CHANGES_REQUESTED** [review]

**Review by jensmaurer — APPROVED** [review]

**Comment by tkoeppe** [comment]
Could you please reorganize (and force-push) these changes to have one for C++23 and one for C++26? I'll apply the C++23 changes to the IS as well.

**Comment by tkoeppe** [comment]
@AlisdairM ^^^

**Pull request merged**